pkg/store/instance.go

```diff
@@ -27,10 +30,12 @@ import (
 type Status = string
 
 const (
-	StatusUnknown Status = ""
-	StatusBroken  Status = "Broken"
-	StatusStopped Status = "Stopped"
-	StatusRunning Status = "Running"
+	StatusUnknown       Status = ""
+	StatusUninitialized Status = "Uninitialized"
+	StatusInstalling    Status = "Installing"
```

Comment:
Installing what?

Reply:
While the hostagent is running `wsl2.initVM` (the `wsl.exe --import ...` command), the status reported by wsl.exe --list -v will be "Installing". I can change the name or just map "Installing" to "Uninitialized" if that makes more sense

![image](https://github.com/lima-vm/lima/assets/424987/80e93ac2-1c7c-4ff6-a2b8-afdd599eb544)